lang/main-bg.json

```diff
@@ -773,7 +785,7 @@
         "header": "Място за помощ"
     },
     "lonelyMeetingExperience": {
-        "youAreAlone": "Вие сте сами в срещата",
+        "youAreAlone": "Вие сте сам в срещата",
```

Comment:
Do you think we can re-phrase this so it's gender-neutral?

Reply:
I haven't checked the other updates for gender-nutral strings, but this was one thing I was trying to achieve when updating the translations, the last time I did it? :)

Reply:
@damencho Thanks for the update! We'll do an iteration to make sure that the strings are gender neutral.